packages/core/directives/control.ts

```diff
@@ -0,0 +1,44 @@
+import {Component, ElementRef, Input, OnChanges, SimpleChanges, ViewEncapsulation} from '@angular/core';
+import {GoogleMapsAPIWrapper} from '../services/google-maps-api-wrapper';
+import {ControlPosition} from '../services/google-maps-types';
+
+/**
+ * AgmMapControl allows to add a custom control to the map
+ *
+ * ### Example
+ *
+ * ```
+ * <agm-control [position]="position" #my-id>
+ *   <div content>
+ *       <!-- my markup -->
+ *   </div>
+ * </agm-control>
+ * ```
+ *
+ */
+@Component({
+  selector: 'agm-control',
+  template: '<ng-content select="[content]"></ng-content>',
+  styles:
+      ['.agm-control {' +
+       '  background: white;' +
+       '  padding: 10px;' +
+       '}'],
+  encapsulation: ViewEncapsulation.None,
+  host: {class: 'agm-control'}
+})
+export class AgmControl implements OnChanges {
+  /* Position of the control */
+  @Input() position: ControlPosition;
```

Comment:
can you fix the comment syntax?

Reply:
wee need to export the ControlPosition in the map types ts file. So everyone can import the specific type in their logic

Reply:
It's already exported in google-maps-types.ts
`export enum ControlPosition {`
Is that what you mean?

Reply:
@jgutix  Im mean this file: https://github.com/SebastianM/angular-google-maps/blob/master/packages/core/map-types.ts